src/repository/db/FriendshipDbRepo.java

```diff
@@ -23,8 +23,30 @@ public FriendshipDbRepo(String url, String username, String password, Validator<
         this.password = password;
         this.val = val;
         this.fshipsTable = fshipsTable;
+
+        String sql = "CREATE TABLE IF NOT EXISTS " + fshipsTable +
+                "(email1 varchar," +
+                " email2 varchar, " +
+                "PRIMARY KEY (email1,email2), " +
+                "FOREIGN KEY (email1) references users(email), " + //ON DELETE CASCADE
+                "FOREIGN KEY (email2) references users(email))";
+
+        try (Connection connection = DriverManager.getConnection(url, username, password)) {
+            PreparedStatement ps = connection.prepareStatement(sql);
+            ps.executeUpdate();
+         } catch (SQLException e) {
+            // daca facem o singura conexiune si in functie de cum se arunca exceptie pun la password - postgres
```

Comment:
mi se pare urat tare asa, daca vr schimb si eu parola la db meu in postgres :)))

Reply:
Am incercat si eu sa mi schimb parola din 0000 in postgres si mi am cam prins urechile =)) Daca tie iti iese mai repede mai degraba schimbi in postgres

Reply:
gata boss